content/prerequisites/cloud9/index.md

```diff
@@ -33,8 +33,7 @@ If you have issues using the Cloud9 environment, please see the [Troubleshooting
 Clone the repository to your working directory. In the Cloud9 terminal run:
 
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-git clone https://github.com/aws-samples/cfn101-workshop
-:::
+git clone https://github.com/aws-samples/cfn101-workshop:::
```

Comment:
```suggestion
git clone https://github.com/aws-samples/cfn101-workshop
:::
```

Reply:
The suggested formatting breaks the generated code block.

Reply:
Running preview_build helped me fix this